src/Mod/Sketcher/App/planegcs/Constraints.cpp

```diff
@@ -2585,5 +2585,112 @@ double ConstraintEqualLineLength::grad(double *param)
     return deriv*scale;
 }
 
+// ConstraintC2CDistance
+ConstraintC2CDistance::ConstraintC2CDistance(Circle &c1, Circle &c2, double *d)
+{
+    this->d = d;
+    pvec.push_back(d);
+
+    this->c1 = c1;
+    this->c1.PushOwnParams(pvec);
+
+    this->c2 = c2;
+    this->c2.PushOwnParams(pvec);
+
+    origpvec = pvec;
+    pvecChangedFlag = true;
+    rescale();
+}
+
+void ConstraintC2CDistance::ReconstructGeomPointers()
+{
+    int i=0;
+    i++; // there is an inline function for distance
```

Comment:
What does this fragment mean?

Reply:
pvec with index 0 is distance ( d at line 2591) it's skipped here as it doesn't need a reconstruction.
This pvec has an getter defined at src/Mod/Sketcher/App/planegcs/Constraints.h:755

i is not initialized at 1 in order to explicit the skip.

@abdullahtahiriyo You confirm ?